doc/en/getting-started.rst

```diff
@@ -185,6 +185,21 @@ You can find out what kind of builtin :ref:`fixtures` exist by typing::
 
     pytest --fixtures   # shows builtin and custom fixtures
 
+Contributing tests to an existing project
```

Comment:
I disagree on the 'people being used to virtualenvs' point, many users coming to a getting started section will not be used to any of the basic concepts of Python or unit testing.  This being too specific/narrow makes sense to me though. It's kinda just tacked on the end at the moment.  

Seems like running the tests on a new project will be enough people's first experience with pytest that there should be a walkthrough for it somewhere, and the getting started section was the first place I looked for it.  Maybe there should be a page on it linked at the bottom?